packages/venia-ui/lib/components/MiniCart2/minicart.css

```diff
@@ -0,0 +1,80 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
+    transition-property: opacity, visibility;
+
+    /* It sits over all background content. */
+    z-index: 3;
+}
+
+.root_open {
+    composes: root;
+
+    opacity: 1;
+    visibility: visible;
+
+    /* It animates to being open. */
+    transition-duration: 224ms;
+    transition-timing-function: var(--venia-anim-in);
+}
+
+.body {
+    margin-top: 1rem;
+    overflow-y: scroll;
+}
+
+.contents {
+    align-content: space-between;
+    background-color: rgb(var(--venia-background-color));
+    border-radius: 1px;
+    box-shadow: 1px 1px 5px rgb(var(--venia-grey-darker));
+    display: grid;
+    /* The header and footer heights may change based on PWA-116 and PWA-601. */
+    grid-template-rows: 2rem 1fr 8rem;
+    /* These heights may change based on PWA-605 and should be based on 1.5 and 2.5 visible items. */
+    height: 100%;
+    min-height: 16rem;
+    max-height: 22.5rem;
+    overflow: hidden;
+    padding: 1rem;
+    position: fixed;
+    right: 0;
+    /* This value is equal to the site header height. */
+    top: 3.5rem;
+    width: 30rem;
+}
+
+.footer {
+    border-top: solid 0.4rem rgb(var(--venia-grey));
+    overflow: hidden;
+}
+
+.header {
+    border-bottom: solid 0.15rem rgb(var(--venia-grey));
+    overflow: hidden;
+}
+
+.mask {
+    /* The mask takes up the entire screen. */
+    position: absolute;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* The mask blurs out everything behind it. */
+    backdrop-filter: blur(50px);
```

Comment:
`backdrop-filter` [isn't supported in Firefox yet](https://caniuse.com/#feat=css-backdrop-filter). We may have to revisit the design because we've said we don't want browser-specific CSS.

Reply:
What was wrong with the mask before? Or did the design specifically call for a blur effect?

Reply:
It specifically called for blur, yeah